src/main/java/org/benetech/servicenet/conflict/detector/AbstractDetector.java

```diff
@@ -17,20 +17,28 @@ protected boolean areEquals(String current, String offered) {
 
     protected List<Conflict> detectConflicts(T current, String name, String name2, String fieldName) {
         List<Conflict> conflicts = new LinkedList<>();
-        if (!this.areEquals(name, name2)) {
+        if (detect(name, name2)) {
             conflicts.add(createConflict(current, name, name2, fieldName));
         }
         return conflicts;
     }
 
+    protected boolean detect(String name, String name2) {
```

Comment:
this method name does not indicate what it does, and it might lead to confusions in the future

Reply:
this is logic is responsible for detecting conflicts in a method called detectConflicts, but this one does not return conflicts as a result. That's why I called it detect

Reply:
it's a boolean method and it's not obvious what the result will be